test/blackbox/api/dds-pim/PubSubReader.hpp

```diff
@@ -456,7 +460,7 @@ class PubSubReader
         bool ret = false;
         do
         {
-            receive_one(datareader_, ret);
+            receive(datareader_, std::ref(ret));
```

Comment:
Why is `std::ref` needed here?

Reply:
It isn't 😅. A side effect of changing one's mind in the middle of a pull request.